Doc/library/exceptions.rst

```diff
@@ -212,6 +212,15 @@ The following exceptions are the exceptions that are usually raised.
    accidentally caught by code that catches :exc:`Exception` and thus prevent
    the interpreter from exiting.
 
+   .. note::
+
+      Catching a :exc:`KeyboardInterrupt` requires special consideration.
+      Because it can be raised at unpredictable points, it may may, in some
```

Comment:
two "may"s here... otherwise looks good to me.

Reply:
whoops. Thanks.